libcontainer/intelrdt/intelrdt.go

```diff
@@ -504,6 +522,20 @@ func (m *Manager) GetPath() string {
 	return m.path
 }
 
+// GetMonPath returns path of the monitoring group of the container. Returns an
+// empty string if the container does not have a individual dedicated
+// monitoring group.
+func (m *Manager) GetMonPath() string {
+	if closPath := m.GetPath(); closPath != "" && m.config.IntelRdt.EnableMonitoring {
```

Comment:
Checking a `bool` is faster so why don't we check it first? Also, to simplify the code:

```go
if !m.config.IntelRdt.EnableMonitoring {
	return ""
}
closPath := m.GetPath()
if closPath == "" {
	return ""
}
...
```